pkg/controller/certificatemanager/certificatemanager_controller.go

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) 2022 Tigera, Inc. All rights reserved.
```

Comment:
If you're adding a controller to this package you'll need to move the "utility" functions somewhere else, like into util under a subpackage. If you don't you're you're breaking the contract that controllers don't access each other packages (and I think the other controllers access this package).